tests/v2/validation/upgrade/workload.go

```diff
@@ -244,17 +244,33 @@ func checkPrefix(name string, prefix string) bool {
 	return strings.HasPrefix(name, prefix)
 }
 
-// validateDaemonset checks daemonset available number is equal to the number of workers in the cluster
+// validateDaemonset checks daemonset available number is equal to the number of workers in the downstream cluster
```

Comment:
nit - can we consolidate these comments? So something like `validateDaemonset checks that the available number of daemonsets equals the number of workers in a downstream cluster and the number of nodes in the local cluster`.

Reply:
yes.  I will consolidate these comments, sure thing 👍

Reply:
updated! ✅